src/main/javacc/copy.jjt

```diff
@@ -0,0 +1,247 @@
+options {
+  MULTI=true;
+  VISITOR=true;
+  NODE_DEFAULT_VOID=true;
+}
+
+PARSER_BEGIN(Copy)
+
+package com.google.cloud.spanner.pgadapter.parsers;
+
+public class Copy {
+
+  public static void parse(String stmt) throws Exception {
+    java.io.InputStream stmtStream = new java.io.ByteArrayInputStream(stmt.getBytes());
+    Copy t = new Copy(stmtStream);
+    try {
+      ASTStart n = t.Start();
+      CopyVisitor v = new CopyDumpVisitor();
+      n.jjtAccept(v, null);
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
+      e.printStackTrace();
+      throw e;
+    }
+  }
+
+}
+
+PARSER_END(Copy)
+
+SKIP :
+{
+  " "
+| "\t"
+| "\n"
+| "\r"
+| <"//" (~["\n","\r"])* ("\n"|"\r"|"\r\n")>
+| <"/*" (~["*"])* "*" (~["/"] (~["*"])* "*")* "/">
+}
+
+TOKEN : /* KEYWORDS */
+{
+  <K_STDIN: "STDIN">
+| <K_STDOUT: "STDOUT">
+| <K_TO: "TO">
+| <K_FROM: "FROM">
+| <K_COPY: "COPY">
+| <K_WITH: "WITH">
+}
+
+TOKEN : /* COPY OPTIONS */
+{
+  <O_BINARY: "BINARY">
+| <O_CSV: "CSV">
+| <O_FORMAT: "FORMAT">
+| <O_FREEZE: "FREEZE">
+| <O_DELIMITER: "DELIMITER">
+| <O_NULL: "NULL">
+| <O_HEADER: "HEADER">
+| <O_QUOTE: "QUOTE">
+| <O_ESCAPE: "ESCAPE">
+| <O_FORCE_QUOTE: "FORCE_QUOTE">
```

Comment:
some of these options, as well as COPY FROM <filename> won't be supported in the first version of COPY. Should we throw a parse exception here if those options are used or handle the exception later on?

Reply:
I can add in an exception for now and later we can add support for them if necessary.